tests/components/recorder/test_init.py

```diff
@@ -135,13 +136,17 @@ def _add_events(hass, events):
         return [ev.to_native() for ev in session.query(Events)]
 
 
-# pylint: disable=redefined-outer-name,invalid-name
-def test_saving_state_include_domains(hass_recorder):
+def test_saving_state_include_domains_globs(hass_recorder):
```

Comment:
We should not alter the behavior of existing tests beyond that is needed to make the function not reject argument changes, it is better to add new ones so we prove the old tests are still working without changes.

Reply:
Reverted changes to tests where I added glob filters and added separate ones specifically for that.